app/helpers/evaluations_helper.rb

```diff
@@ -3,9 +3,9 @@
 # View helpers for calculating evaluation & submission details.
 module EvaluationsHelper
   STATUS_COLORS = {
-    not_started: 'bg-error-dark',
-    in_progress: 'bg-accent-warm-dark',
-    completed: 'bg-success-dark',
+    not_started: 'bg-red-vivid-60v',
+    in_progress: 'bg-orange-warm-vivid-50v',
+    completed: 'bg-green-cool-vivid-60v',
```

Comment:
these colors were just updated to the correct values globally [here](https://github.com/GSA/Challenge_platform/pull/444/files), were these values changed again in the design?

Reply:
as far as I can tell, they are the same values that are already there, so let's revert this to the theme colors from before since it is easier to reuse those colors throughout the app.
```suggestion
    not_started: 'bg-error-dark',
    in_progress: 'bg-accent-warm-dark',
    completed: 'bg-success-dark',
```

Reply:
Yep, the colors were updated in the new figma design ([evaluator submission evaluation status](https://www.figma.com/design/sLkJM1Ua9Zu3HW2h4mVdgr/Challenge.gov-Eval-(Updated-01%2F27%2F25)?node-id=7003-160967&t=9B4OVFh82DUStepA-0) & [evaluation status key](https://www.figma.com/design/sLkJM1Ua9Zu3HW2h4mVdgr/Challenge.gov-Eval-(Updated-01%2F27%2F25)?node-id=7003-153484&t=9B4OVFh82DUStepA-0)). I asked Renata about the colors on Monday and she confirmed these are the new colors for the evaluation statuses

Reply:
@stepchud `green-cool-vivid-60v` and `green-cool-60v` are [different colors](https://uswds-tailwind.com/colors) and so are `bg-orange-warm-vivid-50v` and `orange-warm-50v`

Reply:
right, what I'm saying is they have already been corrected to the new values (just last week). the changes on this PR don't actually affect the color rendered in the page because they were already correct.

Reply:
however I do see the recused evaluator's name on the submissions list is a slightly different orange color (`bg-error`). Renata wants us to remove the highlight from that name though so that won't be an issue. is that being updated on any of your PRs? sorry I'm still trying to catch up with all the feedback I'm getting. 😕 

![Screenshot 2025-03-05 at 9 57 28 AM](https://github.com/user-attachments/assets/4908ee85-4bbe-4ef3-a86a-4ceb94858c1c)

Reply:
Okay cool - I understand, I'll revert that change! Thanks 👍

And yep, I removed that highlight from the name in [the PR #399](https://github.com/GSA/Challenge_platform/pull/461)

Reply:
Just reverted that color change - my bad, thanks for double checking that! @stepchud